### ISSUE TITLE
Create District Monitoring dashboard

### ISSUE THREAD

**Issue by mmannion** [NWFWMD-IT/Hydrologic-Monitoring-Project#202]
Waiting on:

- [x] Last visited view < ArcGIS Enterprise upgrade
- [x] Project Numbers

# Tasks

- [x] @mmannion Add column `LocationIssue.IsActive`
- [x] @mmannion Create view that shows Locations with active LocationIssue
  - Include LocationIssue attributes
- ~@SarahNWF Research synchronizing active record between related dashboards~
  - ~Hyperlink URL in view~
  - ~Experience Builder search widget across multiple dashboards~
- ~@SarahNWF Design solution for deactivating Location Issues once they've been resolved~ Moved to #256
  - ~E.g. in dashboard, or something easier than Pro~
- [x] @john-beamish Create Dashboard with the following features
  - Filter by 
    - [x] Active Monitoring Type
    - [x] County
    - [x] 'Active' Sensors and Gages 
    - [x] Assigned To (Staff/ Quarterly GWL run name)
    - [x] Project number
    - [x] NWFID
    - [x] Location name
    - [x] Telemetry address (IP/ GOES address)
    - [x] Visit Frequency
    - [x] Last visit date
   - [x] Display multiple Assignments at the same time in different symbology/ colors to see if current distribution makes sense
   - [x] Display location issue by symbology/ color based on type of issue ~with the ability to turn off issue when it's been addressed~
   - [x] Display locations that have been visited in the current calendar month (works with assignment filter)
   - [ ] ~Display locations that are overdue based on visit frequency, including a feature that shows a warning for which sites have not been visited at the 3rd week of the expected visit month (works with assignment filter)~
     - Superseded by #262

**Comment by mmannion** [comment]
As part of schema updated 2:
- Add column `LocationIssue.IsActive`
  - Populate all current (i.e. non-archived) rows with 'Yes'
- Add column `LocationLastVisited.IssueCount`
  - Integer column that contains count of current Location Issue records with `IsActive = 'Yes'`

I decided to reuse the existing `LocationLastVisited` view and indicate the presence of Location Issues using an integer count, absent more specific directions. These are both easy to change, though - names, values, data types, separate views, etc.

**Whatever is most useful for the dashboard is easy to change here.**

Therefore, please ask if you'd like to consume the Location Issue data some other way; it'll be better to adapt the view to meet the needs of the dashboard than vice versa.

A log of the schema update follows:

```
2025-04-28 23:03:51.618 DEBUG     Configuring log file C:\users\hydro\documents\update_database_hydro.2.2025-04-28_3.log
2025-04-28 23:03:53.661 DEBUG     Adding log FileHandler
2025-04-28 23:03:53.678 DEBUG     Setting log FileHandler Formatter
2025-04-28 23:03:53.679 DEBUG     Adding FileHandler to Logger
2025-04-28 23:03:53.682 INFO      ================================================================================
                                  Hydrologic Data Model Geodatabase Update 2
                                  --------------------------------------------------------------------------------
                                  Target database server:  sql-gis
                                  Log level:               DEBUG
                                  Log file:                C:\users\hydro\documents\update_database_hydro.2.2025-04-28_3.log
                                  ================================================================================
2025-04-28 23:03:53.685 INFO      Connecting to geodatabase
2025-04-28 23:03:53.697 DEBUG     Checking OS username
2025-04-28 23:03:53.702 DEBUG     Created temporary directory C:\Users\hydro\AppData\Local\Temp\tmplvbuu6ka
2025-04-28 23:03:58.552 DEBUG     Created database connection file: C:\Users\hydro\AppData\Local\Temp\tmplvbuu6ka\connection.sde
2025-04-28 23:03:58.553 DEBUG     Validating geodatabase connection
2025-04-28 23:03:58.796 DEBUG     Describe type: Workspace
2025-04-28 23:03:58.798 INFO      Updating table: LocationIssue
2025-04-28 23:03:58.804 INFO            Adding columns
2025-04-28 23:03:58.806 INFO                    IsActive
2025-04-28 23:04:00.496 INFO      Updating table: LocationVisit
2025-04-28 23:04:00.498 INFO            Dropping columns
2025-04-28 23:04:00.508 INFO                    BatteryNeedsReplacement
2025-04-28 23:04:01.809 INFO      Updating table: RainfallTips
2025-04-28 23:04:01.810 INFO            Dropping columns
2025-04-28 23:04:01.817 INFO                    FalseTipRemoved
2025-04-28 23:04:02.430 INFO            Dropping view
2025-04-28 23:04:04.574 INFO            Creating view
2025-04-28 23:04:04.575 INFO                    Creating view LocationLastVisit
2025-04-28 23:04:06.476 INFO                            Registering with geodatabase
2025-04-28 23:04:08.47  INFO                            Setting view alias
2025-04-28 23:04:08.143 INFO                            Setting field aliases
2025-04-28 23:04:08.145 INFO                                    ID > Geodatabase Unique ID
2025-04-28 23:04:08.572 INFO                                    NWFID > NWFID
2025-04-28 23:04:08.881 INFO                                    Name > Name
2025-04-28 23:04:09.146 INFO                                    LastVisit > Last Visit
2025-04-28 23:04:09.474 INFO                                    IssueCount > Issue Count
2025-04-28 23:04:09.787 INFO                            Assigning privileges
2025-04-28 23:04:09.789 INFO                                    HQ\arcgis      GRANT     GRANT
2025-04-28 23:04:10.122 INFO      Done.
```

**Comment by kprice-TLH** [comment]
Steve and Katie discussed the features needed for the dashboard. Below is a list
[] Filter by
  [] County
